src/ratesEngine.ts

```diff
@@ -40,7 +40,11 @@ export async function ratesEngine(): Promise<void> {
     } else {
       bookmark = undefined
     }
-    asDbQueryResult(result)
+    try {
+      asDbQueryResult(result)
```

Comment:
asDbQueryResult cleaner should be asArray(asUnknown), then clean each transaction individually